metadata-models/src/main/pegasus/com/linkedin/common/BaseFieldMapping.pdl

```diff
@@ -0,0 +1,21 @@
+namespace com.linkedin.common
+
+/**
+ * Base model representing field mappings
+ */
+record BaseFieldMapping {
+  /**
+   * Audit stamp containing who reported the field mapping and when
+   */
+  created: optional AuditStamp
+
+  /**
+   * Type of the transformation involved in generating destination field from source fields. e.g., BLACKBOX, IDENTITY, CUSOTM
```

Comment:
typo in CUSTOM. See comment below on renaming it to UDF.

Reply:
As discussed, Moved to creating individual models for types. This can help scale each of the individual models.